pkg/codegen/codegen.go

```diff
@@ -298,6 +309,38 @@ func GenerateTypeDefinitions(t *template.Template, swagger *openapi3.Swagger, op
 	return typeDefinitions, nil
 }
 
+// Generates operation ids, context keys, paths, etc. to be exported as constants
+func GenerateConstants(t *template.Template, ops []OperationDefinition) (string, error) {
+	var buf bytes.Buffer
+	w := bufio.NewWriter(&buf)
+
+	constants := Constants{
+		SecuritySchemeProviderNames: []string{},
+	}
+
+	providerNames := map[string]struct{}{}
```

Comment:
We've been using map[string]bool for a map of key presence. It allows for `if foo[x]`. It's nice to stick to one style.